src/aeso_icode.erl

```diff
@@ -103,6 +106,10 @@ new_env() ->
 set_name(Name, Icode) ->
     maps:put(contract_name, Name, Icode).
 
+-spec set_payable(boolean(), icode()) -> icode().
+set_payable(Name, Icode) ->
```

Comment:
`Name` is a boolean?

Reply:
Copy pasted, should be `Payable` of course.